guides/release/services/index.md

```diff
@@ -140,3 +140,145 @@ Note `cart` being used below to get data from the cart.
 ```
 
 <!-- eof - needed for pages that end in a code block  -->
+
+### Accessing services from native classes
+
+If you want to access a service from a plain JavaScript class, you'll need to get a reference to the "[owner](https://api.emberjs.com/ember/release/modules/@ember%2Fowner)" object, which is responsible for managing services.
+
+First, we can define a class that accesses services as described above:
+
+```javascript {data-filename=app/components/cart-contents/vanilla-class.js}
+import { service } from '@ember/service';
+
+export class VanillaClass {
+  @service shoppingCart;
+
```

Comment:
If the class is going to use a service, it really should have the owner passed in to the constructor and do the setOwner in the constructor like glimmer components imo

Reply:
I don't really agree :-\\ 

if the responsibility of setting up the owner is always in the constructor then it's impossible to abstract reasonably, as one might to do here: https://ember-resources.pages.dev/funcs/link.link